integ-tests/src/main/java/com/google/testdata/PerformanceTestDataHelper.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.testdata;
+
+import static com.google.base.TestConstants.SQL_TESTDATA_BASE_PATH;
+import static com.google.sql.SqlHelper.executeQueries;
+import static com.google.sql.SqlHelper.getSql;
+import static com.google.testdata.TestDataHelper.getRandomDbName;
+import static com.google.testdata.TestDataHelper.getRandomTableName;
+import static java.lang.String.format;
+import static java.lang.System.lineSeparator;
+
+import com.google.common.base.Joiner;
+import com.google.testdata.pojo.TestDataUser;
+import com.google.testdata.utils.PerformanceTestDataGeneratorQueryExecutor;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.concurrent.ForkJoinPool;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** Helper class providing test data generation methods */
+public final class PerformanceTestDataHelper {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(PerformanceTestDataHelper.class);
+
+  private PerformanceTestDataHelper() {}
+
+  /**
+   * @param connection DB connection parameter
+   * @param tablesCount Count of tables to generate
+   */
+  public static void generateTables(Connection connection, int tablesCount) throws SQLException {
+    final int USER_COUNT = 5;
+    final int TABLES_PER_USER = tablesCount / USER_COUNT;
+    final int SIZE = 2048 * TABLES_PER_USER;
+
+    HashMap<TestDataUser, List<String>> performancePayload = new HashMap<>();
+    List<String> preconditionQueries = new ArrayList<>();
+
+    final String performancePrecondition1 = getSql(SQL_TESTDATA_BASE_PATH + "users_data.sql");
+    final String performancePrecondition2 =
+        getSql(SQL_TESTDATA_BASE_PATH + "performance_data_1.sql");
+    final String performancePrecondition3 =
```

Comment:
nit: it seems they're only used once. Avoid using one-time variables since they're not complex expressions.

If you expect to have more calculations based on them in the future, then them meaningful names like
PreconditionForCreateUser
PreconditionForCreateDatabase
PreconditionForGrantCreateTable

Reply:
Please address unresolved comments, thanks.